src/ProtocolGateway.Core/Instrumentation/IPerformanceCounterManager.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.ProtocolGateway.Instrumentation
+{
+    public interface IPerformanceCounterManager
+    {
+        SafePerformanceCounter GetCounter(string category, string name);
```

Comment:
therefore SPC can be removed from net standard version.

---
In reply to: [114232977](https://github.com/ancaantochi/azure-iot-protocol-gateway/pull/1#discussion_r114232977) [](ancestors = 114232977)